.ci.yaml

```diff
@@ -2175,7 +2175,7 @@ targets:
     properties:
       tags: >
         ["devicelab", "android", "linux", "pixel", "7pro"]
-      task_name: external_ui_integration_test
+      task_name: external_textures_integration_test
```

Comment:
I believe this is only cosmetic, but I'd suggest keeping the `task_name` and the `name` field above matching. So the `name` above on line 2171 would change to `Linux_pixel_7pro external_textures_integration_test`.

Reply:
Unfortunately I can't do that in a single PR because you can only rename a task by starting it as bringup: true. I can do that but I'll need a few more PRs.